Sources/PubNub/Networking/Routers/SubscribeRouter.swift

```diff
@@ -594,12 +598,16 @@ enum MessageActionEventType: String, Codable, Hashable {
 public struct MessageActionEvent: Codable, Hashable {
   public let type: String
   public let value: String
+  public fileprivate(set) var uuid: String = ""
+  public fileprivate(set) var channel: String = ""
   public let actionTimetoken: Timetoken
   public let messageTimetoken: Timetoken
 
-  public init(type: String, value: String, actionTimetoken: Timetoken, messageTimetoken: Timetoken) {
+  public init(type: String, value: String, uuid: String, channel: String, actionTimetoken: Timetoken, messageTimetoken: Timetoken) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line should be 120 characters or less: currently 132 characters](https://app.codacy.com/manual/PubNub/swift/pullRequest?prid=5482374)